jboss-kie-kieserver/added/launch/kieserver-verify.sh

```diff
@@ -10,6 +10,7 @@ setKieEnv
 dumpKieEnv
 
 function verifyServerContainers() {
+    echo "$JAVA_OPTS_APPEND"
```

Comment:
what is the point of add it here?

Reply:
Because the JAVA_OPTS_APPEND could be used t owhom need to read those values, if is move inside the if it will be used only if KIE_SERVER_CONTAINER_DEPLOYMENT isn't empty.

Reply:
Moved